doc/modules/cross_validation.rst

```diff
@@ -719,8 +719,9 @@ group information can be used to encode arbitrary domain specific pre-defined
 cross-validation folds.
 
 Each training set is thus constituted by all the samples except the ones
-related to a specific group. This is basically the same as
-:class:`LeavePGroupsOut` with ``n_groups=1``.
+related to a specific group. This is the same as :class:`LeavePGroupsOut` with
+``n_groups=1`` and the same as :class:`GroupKFold` in the case where
+`n_splits` is equal to the number of groups
```

Comment:
Nit: Using "with" here is more consistent with the first part of the sentence which says "with n_groups=1"

```suggestion
`n_groups=1` and the same as :class:`GroupKFold` with `n_splits` equal
to the number of groups.
```